src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -0,0 +1,437 @@
+    <div {{ block( 'field_attributes' ) }}>
+    {% autoescape false %}
+    {% if type == "html5_video"
+        or type == "quick_time"
+        or type == "windows_media_player"
+        or type == "real_player" %}
+        <video src="{{ download }}" width="50%" height="100%" controls>
+            {{ 'ezmedia.browser_does_not_support_html5_video'|trans|desc('Your browser does not support HTML5 video') }}
+        </video>
+        <table>
+            <thead>
+                <tr>
+                    <th>{{ 'ezmedia.video_file_properties'|trans|desc('Video file properties') }}:</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>{{ 'ezmedia.file_name'|trans|desc('File name') }}:</td>
+                    <td>{{ value.fileName }}</td>
+                </tr>
+                <tr>
+                    <td>{{ 'ezmedia.type'|trans|desc('Type') }}:</td>
+                    <td>{{ value.mimeType }}</td>
+                </tr>
+                <tr>
+                    <td>{{ 'ezmedia.size'|trans|desc('Size') }}:</td>
+                    <td>{{ value.fileSize|ez_file_size( 1 ) }}</td>
+                </tr>
+                <tr>
+                    <td>{{ 'ezmedia.display_controls'|trans|desc('Display controls') }}:</td>
+                    <td>{{hasController}}</td>
+                </tr>
+                <tr>
+                    <td>{{ 'ezmedia.auto_play'|trans|desc('Auto play') }}:</td>
+                    <td>{{autoplay}}</td>
+                </tr>
+                <tr>
+                    <td>{{ 'ezmedia.loop'|trans|desc('Loop') }}:</td>
+                    <td>{{loop}}</td>
+                </tr>
+            </tbody>
+        </table>
+    {% endif %}
+    {% endautoescape %}
+    </div>
+{% endspaceless %}
+{% endif %}
+{% endblock %}
+
+{% block ezobjectrelationlist_field %}
+{% spaceless %}
+    {% if not ez_is_field_empty( content, field ) %}
+    {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ez-field-preview ez-field-preview--ezobjectrelationlist')|trim}) %}
+    <div {{ block( 'field_attributes' ) }}>
+        <div class="ez-table-header ">
+            <div class="ez-table-header__headline">{{ 'ezobjectrelationlist.multiple_relations'|trans|desc('Multiple relations') }}:</div>
+        </div>
+        <table class="table">
+            <thead>
+                <tr>
+                    <th>{{ 'ezobjectrelationlist.name'|trans|desc('Name') }}</th>
+                    <th>{{ 'ezobjectrelationlist.content_type'|trans|desc('Content Type') }}</th>
+                    <th>{{ 'ezobjectrelationlist.created'|trans|desc('Created') }}</th>
+                </tr>
+            </thead>
+            {% for contentId in field.value.destinationContentIds %}
+            <tr>
+                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'viewType': 'embed', 'noLayout': 1, 'layout': 'preview_ezobjectrelationlist_row'})) }}
```

Comment:
I think this should be `layout: false`, rather than `noLayout: 1`

Reply:
Does not matter as the twig used for rendering ezobjectrelationlist isn't extending pagelayout. But I will fix it nonetheless.